ios/Video/NowPlayingInfoCenterManager.swift

```diff
@@ -226,6 +226,12 @@ class NowPlayingInfoCenterManager {
             })
         }
 
+        if let urlAsset = currentItem.asset as? AVURLAsset, urlAsset.url.absoluteString.hasSuffix("m3u8") {
```

Comment:
You cannot consider all m3u8 as live, please a a dedicated flag in the metadata api to set this flag

Reply:
I think we can check with `CMTIME_IS_INDEFINITE(_playerItem.asset.duration)`